credentials/src/test/kotlin/web5/sdk/credentials/StatusListCredentialTest.kt

```diff
@@ -27,17 +27,8 @@ class StatusListCredentialTest {
       specExampleRevocableVcText
     )
 
-    assertTrue(
-      specExampleRevocableVc.vcDataModel.contexts.containsAll(
-        listOf(
-          URI.create("https://www.w3.org/ns/credentials/v2"),
-          URI.create("https://www.w3.org/ns/credentials/examples/v2")
-        )
-      )
-    )
```

Comment:
Should this be removed? Seems unrelated to the test vectors on first glance.

Reply:
I removed it because we were creating a VC that was following the 2.0 model, but we're implementing the 1.1 version of VCs. 

It goes hand in hand with the changes to the `revocable_vc.json` file.